src/Psalm/ErrorBaseline.php

```diff
@@ -286,7 +286,7 @@ function (string $extension) : string {
         $xml = preg_replace_callback(
             '/<files (psalm-version="[^"]+") (?:php-version="(.+)"(\/?>)\n)/',
             /**
-             * @param array<int, string> $matches
+             * @param string[] $matches
```

Comment:
I don't think you need to change this, `array<int, string>` should be accepted whenever `string[]` is expected. (`string[]` is an alias of `array<string>` itself an alias of `array<array-key, string>` which is a parent type of `array<int, string>`

Reply:
You do need to change this, because `callable(array<array-key, string>)` is a *subtype* of `callable(array<int, string>)` due to parameter contravariance.

Reply:
Afaik the change is correct, and equivalent to this:

```suggestion
             * @param array<array-key, string> $matches
```

Since the `$search` parameter already uses `string[]` though, I would strongly recommend to use `string[]` as well, to be consistent.